### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD019.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#24] [General]
# https://blog.3place.cn/3place/fsd/FSD019.html

https://blog.3place.cn/3place/fsd/FSD019.html

**Comment by picklecai** [comment]
跟你在一起的那个人
我从心底里觉得可憎
在我整个一生里面
没见过像他那讨厌的脸
使我觉得心如刀扎。

与其说这是女人的直觉，不如说这是天使的直觉。玛加蕾特作为一个纯洁的少女，还保留了天使的天赋。就像孩童、婴儿保留了人善良的本能一样。

**Comment by picklecai** [comment]
他每次一走进门
总是那样瞧不起人
半带着愤怒
看来他对任何事都不感兴趣
从他脸上看得分明
他是不会爱任何人

原来魔鬼就是这样的：失去爱的能力，失去兴趣（好奇心）。

**Comment by picklecai** [comment]
今夜我乐愿为你不上门闩；

少女的心，悄悄地打开了。
开门本身，就是幸福的一部分。

**Comment by picklecai** [comment]
感情最要紧
名称不过是笼罩
天火的声响和烟雾。

看本质。
